exercises/isbn-verifier/canonical-data.json

```diff
@@ -117,6 +117,14 @@
               "isbn": ""
             },
             "expected": false
+        },
+        {
+            "description": "input is 9 characters",
+            "property": "isValid",
+            "input": {
+                "isbn": "134456729"
```

Comment:
indentation interesting, but since we are not enforcing it automatically, my only argument is "it should be the same so that it is consistent, which will maybe make it easier to read"